### PR TITLE
Replaced screenshot with copyable commands

### PR DESCRIPTION
**Pull request by eleonorarocchi**

### CODE REVIEW

**Comment by shmool** [comment]
Thanks @eleonorarocchi for the pull request, and the willingness to improve the tutorial :)
I was thinking that the screenshot is more useful in this case for several reasons:
- The commands may change (for example, the command to change the branch name is reletavely new)
- There are variables that are part of the command, like you have noted the `[USERNAME]` and same for the git repository (the reader may have chosen a different name)
- GitHub gives a convenient way to copy the commands with the variables already assigned correctly
- By copying from GitHub and not from the tutorial, the reader learnes and practices how to do this for any project.

What are your thoughts about this?

**Comment by eleonorarocchi** [comment]
Yes you are right, then I would recommend leaving the screenshot but
writing a note repeating these points you listed for example, to give an
indication. Above all, the screenshot writes to click on the copy button
(and obviously it doesn't work being a screenshot) but it is misleading in
my opinion.

Il giorno lun 3 ott 2022 alle ore 22:04 Shmuela Jacobs <
***@***.***> ha scritto:

File: workshop-todo-list/appendix-1-git-and-github.md [hunk 1]
```diff
@@ -36,7 +36,11 @@ Don't add any templates or files. **Click on the button "Create repository".**
 
 You have several options to set up the repository. You have already created the project on your computer, so the relevant option is to **push an existing repository from the command line**. Copy the commands starting with `git` \(you can click the "copy" button on the right side to copy all the commands\) and paste into the terminal that's open in your project folder. \(You don't have to paste line by line. You can copy all three lines and paste them. The commands will run one after the other, except the last one where you'll have to hit Enter.\)
 
-![The commands to connect and push your project to the repository](../.gitbook/assets/image%20%287%29.png)
+```text
+git remote add origin git@github.com:[USERNAME]/ng-girls-todo-list.git
+git branch -M main
+git push -u oridin main
```
Comment (shmool) [inline workshop-todo-list/appendix-1-git-and-github.md]: plz fix typo in `origin`

**Comment by shmool** [comment]
Awesome, 
Can you please add back the image, fix the typo in the code, and add a suitable description?
(Instruction to click on the icon on GitHub OR copy the lines here + replace the `[USERNAME]` )
Thanks!

**Comment by eleonorarocchi** [comment]
Done!

**Pull request merged**